picard/tagger.py

```diff
@@ -291,6 +293,24 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         if self.autoupdate_enabled:
             self.updatecheckmanager = UpdateCheckManager(parent=self.window)
 
+        from queue import Queue
+        self._pending_files = {}
+        self._pending_files_process_queue = mp.Queue()
+        self._pending_files_results_queue = mp.Queue()
+        self._pending_load_finished_queue = Queue()
+        self._file_loaded_queue = Queue()
+        self._process_stopping = mp.Value('b', False)
+        self._loader_process = mp.Process(target=File._load_check_metadata,
+                                          args=(self._process_stopping,
+                                                self._pending_files_process_queue,
+                                                self._pending_files_results_queue))
+        self._loader_process.start()
+
+        import threading
```

Comment:
`Thread` is already imported at the top, so this import can be removed and `Thread` used directly below.

Reply:
My bad.